src/dc_receive_imf.rs

```diff
@@ -3040,6 +3058,113 @@ mod tests {
         assert_eq!(chat.name, "Atlas Obscura");
     }
 
+    #[async_std::test]
+    async fn test_dhl_mailing_list() {
+        let t = TestContext::new_alice().await;
+        t.set_config(Config::ShowEmails, Some("2")).await.unwrap();
+
+        dc_receive_imf(
+            &t,
+            br#"Message-ID: <123456789.1234567.1234567891234@rnd-17.broadmail.live>
+MIME-Version: 1.0
+Content-Type: multipart/signed; protocol="application/pkcs7-signature"; micalg=sha-256;
+	boundary="----=_Part_1234567_123456789.1614343941878"
+Date: Fri, 26 Feb 2021 13:52:21 +0100 (CET)
+From: DHL Paket <noreply.packstation@dhl.de>
+Reply-To: DHL Paket <noreply.packstation@dhl.de>
+To: alice@example.org
+Subject: Ihr Paket ist in der Packstation 123
+List-Id: <1234ABCD-123LMNO.mailing.dhl.de>
+List-Unsubscribe: <mailto:listoff-1234XYZA-1234ABCD-123111@mailing.dhl.de?subject=unsubscribe>,<https://mailing.dhl.de/go/11/1234XYZA-1234ABCD-1234567-123456-UL.html>
+List-Unsubscribe-Post: List-Unsubscribe=One-Click
+
+------=_Part_1234567_123456789.1614343941878
+Content-Type: multipart/alternative;
+	boundary="----=_Part_2345678_234567890.1614343941876"
+
+------=_Part_2345678_234567890.1614343941876
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: quoted-printable
+Content-Disposition: inline
+
+bla bla
+------=_Part_2345678_234567890.1614343941876
+Content-Type: text/html; charset=UTF-8
+Content-Transfer-Encoding: quoted-printable
+Content-Disposition: inline
+
+<html>bla bla!</html>
+------=_Part_2345678_234567890.1614343941876--
+
+------=_Part_1234567_123456789.1614343941878
+Content-Type: application/pkcs7-signature; name=smime.p7s; smime-type=signed-data
+
+...
+------=_Part_1234567_123456789.1614343941878--
```

Comment:
I think that most test email sources that are that long were put into separate files; doesn't mean I think you have to do this too, just wanted to note it :wink:

Reply:
i also thought about that, however, i think the test is easier to understand if you see the source directly.

and the email source is not that long imho, i removed several longer parts (leaving parts in that may be of interest and are not too long however)

Reply:
but yes, probably makes sense to move that to separate files. i do a commit.